openslides/users/views.py

```diff
@@ -37,10 +38,12 @@ def check_view_permissions(self):
         """
         Returns True if the user has required permissions.
         """
+        is_operator = (str(self.request.user.pk) == self.kwargs.get('pk'))
```

Comment:
Let us please use a consistent solution:
* 'list', 'retrieve' use only check_permission. There we set the result always to True. The permission check is then done in get_restricted_data which is already implemented.
* 'metadata' is only for users with can_see_name
* 'update' and 'partial_update' is for all users that are logged in (see e. g. some lines in core/views.py as example); we check the permissions all in the update() method below.